lib/generators/draper/templates/application_decorator.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationDecorator < Draper::Decorator
```

Comment:
I agree your opinion too :)
However,  how do you think about leaving as comment to notice function of `delegate_all`? And additionally, if you agree with me, I want to add `include Draper::LazyHelpers`.